src/advanced-topics/how-it-works.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Low level details about the framework
+description: Low level details about the framework.
```

Comment:
I personally think that titles shouldn't have full stops at the end.

Reply:
Okay, I just noticed that some did and some didn't. Should I remove all of them across all pages?

Reply:
I think that would be good. The documentation is a bit message at the moment. 
You might want to check with @jstarry before proceeding.
I'm pushing for us to adopt a formal style guide so that we have a definitive authority on some of these things.

Reply:
@dirkpuge I personally think that period's aren't appropriate here because they show up on Gitbook as a subtitle. Thanks for pushing for consistency!

Reply:
@jstarry I understand, I went through and removed the periods

Reply:
By the way, @dirkpuge you might find it easier to use a local editor (on your machine) rather than editing through the Github interface (sorry if I'm assuming correctly – I mean this only in a positive sense, not trying to blame you for anything or anything like that) because you can edit a number of different files on your machine and then create a commit once you've finished editing all of them instead of what you have to do in the Github interface where you create a commit every time you want to "save" a file.

Reply:
@teymour-aldridge Thank you for the advice, I am very new to this so I appreciate the feedback

Reply:
You're most welcome. Feel free to reach out if you need any help :)
p.s. we also have a Discord server, where you can ask for help with Yew if you run into problems